pkg/segment/structs/segmeta.go

```diff
@@ -24,8 +24,9 @@ import (
 const MAX_SEGMETA_FSIZE = 10_000_000 // 10 MB
 
 type ColSizeInfo struct {
-	CmiSize uint64 `json:"cmiSize"`
-	CsgSize uint64 `json:"csgSize"`
+	CmiSize            uint64 `json:"cmiSize"`
+	CsgSize            uint64 `json:"csgSize"`
+	ConsistentCvalSize uint32 `json:"cValSize"` // The size of the column value, given that the size is consistent across all records. The value is set to `1MB` if the column values length is not consistent.
 }
```

Comment:
Using 1MB seems quite odd to me. Why not use 0 or the max uint32 value to indicate this special case?

Alternatively, maybe it would make sense to use a `utils.Option[uint32]` here instead (although with that I think you'd have to write the JSON encode/decode functions, which may cause problems for non-literal types, but perhaps you could only handle certain types like uint32 for now)

This `ColSizeInfo` is JSON encoded and then written to segmeta.json, so whatever you do, we're going to have to support that for a long time

Reply:
Before we decided to use `1MB` because as we will not allow the value size more than `65_535` bytes. So the size will never reach `1MB`. So Should I continue using the same or change it to max of `uint32`?

Reply:
Let's use the max `uint32`